sql/updates/world/2015_02_13_00_world.sql

```diff
@@ -0,0 +1,19 @@
+DROP TABLE IF EXISTS `version_updates`;
+CREATE TABLE `version_updates` (
+	`name` VARCHAR(200) NOT NULL,
+	`timestamp` TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP,
```

Comment:
recommending additional checksum of update file content to detect sneakily changed existing sqls